R/translate-sql-string.R

```diff
@@ -0,0 +1,9 @@
+# R prefers to specify start / stop or start / end
```

Comment:
To fix the collate problem, you need to add a line to the top of `backends-.R`. This is a weird design that I now regret, but I can't figure out how to figure it in a backward compatible way.